### ISSUE TITLE
`GenerateManualPlugin` is not available as package plugin

### ISSUE THREAD

**Issue by MarcoEidinger** [apple/swift-argument-parser#455]
**ArgumentParser version:** `1.1.3`
**Swift version:** 5.6

### Checklist
- [X] If possible, I've reproduced the issue using the `main` branch of this package
- [X] I've searched for [existing GitHub issues](https://github.com/apple/swift-argument-parser/issues)

## Relates to

ArgumentParser includes a SwiftPM plugin for generating man pages. Explore the functionality and configuration by running `swift package plugin experimental-generate-manual --help` from your package root. (https://github.com/apple/swift-argument-parser/pull/332)

### Steps to Reproduce
Add `ArgumentParser` as a package dependency to an existing package

```Swift
    dependencies: [
        // other dependencies
        .package(url: "https://github.com/apple/swift-argument-parser", from: "1.1.3"),
    ],
```

Run command to list available plugins

```
swift package plugin --list
```

### Expected behavior

Expected that `GenerateManualPlugin` is shown as available plugin

```
‘experimental-generate-manual’ (plugin ‘GenerateManualPlugin’ in package ‘swift-argument-parser’)
```

### Actual behavior

No plugin(s) are listed

**Comment by MarcoEidinger** [comment]
@rauhul I am so happy to see that fix merged! What is the ETA for a new release (1.1.4) that will include the fix? After all, the feature to generate man pages is otherwise only working for people pinning to the main branch.

**Comment by MarcoEidinger** [comment]
@natecook1000 what is the ETA for a new release (1.1.4) that will include the fix?
